charts/ndb-operator/README.md

```diff
@@ -0,0 +1,171 @@
+# Nutanix Database Service Operator for Kubernetes
+The NDB operator brings automated and simplified database administration, provisioning, and life-cycle management to Kubernetes.
+
+---
+
+## Pre-requisites
+1. NDB [installation](https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Era-User-Guide-v2_4:top-era-installation-c.html).
+2. A Kubernetes cluster.
+3. Helm installed.
+
+## Installation and Running on the cluster
+Deploy the operator on the cluster:
+```sh
+helm repo add nutanix https://nutanix.github.io/helm/
+
+helm install ndb-operator nutanix/ndb-operator --version 0.0.2
```

Comment:
let's remove the version from the install and install it in a dedicated namespace

`helm install ndb-operator nutanix/ndb-operator -n ndb-operator --create-namespace`